### ISSUE TITLE
loss of snapshot configuration (ddConfig) upon imem restart in dev environment 

### ISSUE THREAD

**Issue by stoch** [K2InformaticsGmbH/imem#259]
Everytime I restart cpro in dev mode (disk db), ddConfig is reset to this:

![image](https://user-images.githubusercontent.com/1941407/57981904-29d69300-7a3e-11e9-87a0-d379f924a2b8.png)

It seems that imem goes into cold start mode:

```
2019-05-19 13:54:03.108 [info] [imem<0.238.0>@imem_seco:start_link:80] [_IMEM_] {imem_seco,80} imem_seco started!
2019-05-19 13:54:03.123 [info] [imem<0.238.0>@imem_domain:start_link:101] [_IMEM_] {imem_domain,101} imem_domain starting...
2019-05-19 13:54:03.123 [info] [imem<0.238.0>@imem_domain:start_link:104] [_IMEM_] {imem_domain,104} imem_domain started!
2019-05-19 13:54:03.139 [info] [imem<0.238.0>@imem_snap:start_link:146] [_IMEM_] {imem_snap,146} imem_snap starting...
2019-05-19 13:54:03.139 [info] [imem<0.546.0>@imem_snap:maybe_coldstart_restore:911] [_IMEM_] {imem_snap,911} Cold Start : auto restoring backup_snapshot_20190518_140051.295039.zip found at c:/projects/git/cpro/snapshot
2019-05-19 13:54:03.186 [info] [imem<0.546.0>@imem_snap:restore_chunk:614] [_IMEM_] {imem_snap,614} restore properties of ddAccount
2019-05-19 13:54:03.279 [info] [imem<0.546.0>@imem_snap:read_chunk:590] [_IMEM_] {imem_snap,590} backup file "snapshot/ddAccount.bkp" restored
2019-05-19 13:54:03.279 [info] [imem<0.546.0>@imem_snap:restore_chunk:614] [_IMEM_] {imem_snap,614} restore properties of ddConfig
2019-05-19 13:54:03.311 [info] [imem<0.546.0>@imem_snap:read_chunk:590] [_IMEM_] {imem_snap,590} backup file "snapshot/ddConfig.bkp" restored
2019-05-19 13:54:03.311 [info] [imem<0.546.0>@imem_snap:restore_chunk:614] [_IMEM_] {imem_snap,614} restore properties of ddRole
2019-05-19 13:54:03.326 [info] [imem<0.546.0>@imem_snap:read_chunk:590] [_IMEM_] {imem_snap,590} backup file "snapshot/ddRole.bkp" restored
2019-05-19 13:54:03.326 [info] [imem<0.546.0>@imem_snap:init:175] [_IMEM_] {imem_snap,175} snapshot directory c:/projects/git/cpro/snapshot
2019-05-19 13:54:03.326 [info] [imem<0.238.0>@imem_snap:start_link:150] [_IMEM_] {imem_snap,150} imem_snap started!
```
This must not happen if the DB on disk is OK.
If I disable snapshotting and then restart the system, snapshotting is enabled again.

**Comment by shamis** [comment]
same as #216 
fix was implemented in #217, but was not merged since this would disable the ability to restore the cluster snapshot for disc nodes. 
It has been agreed to update the start.sh to set `cold_start_recover` to **false**.
This will disabled `cold_start_recover` in dev mode

**Comment by shamis** [comment]
fixed in https://github.com/K2InformaticsGmbH/imem/commit/2773be988e4243d171f3a51df44f403034ac0200
